src/main/scala/org/scalacheck/Prop.scala

```diff
@@ -66,7 +66,7 @@ sealed abstract class Prop extends Serializable { self =>
     this
   )
 
-  /** Convenience method that checks this property with the given parameters
+  /** Convenience method that checks this property
```

Comment:
Maybe it should read "check this property with the [default] parameters"?

Reply:
Maybe. I noticed the link to the `[default] parameters` on line 73, so I thought the original author preferred to highlight it separately. Thus, I made it consistent with the docs of the next method (on line 77).

I'm happy to change it if you wish, but then I think I should change lines 69, 73, and 77.

Reply:
Probably fine to mention it uses the default parameters _and_ links to the default parameters later.